### PR TITLE
LP-2359 Allow learner and peers to complete remaining ORA steps, once ORA is auto scored

### PR DESCRIPTION
**Pull request by zahid-arbisoft**
### Description

[LP-2359](https://philanthropyu.atlassian.net/browse/LP-2359) Allow learner and peers to complete remaining ORA steps, once ORA is auto scored

### Notes

Unit test are updated and all unit test are passing

### CODE REVIEW

**Review by muak50 — APPROVED** [review]
LGTM 👍

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: openedx/features/assessment/helpers.py [hunk 1]
```diff
@@ -127,6 +127,9 @@ def autoscore_ora_submission(submission):
     Auto score ORA submission, by Philu bot. Score will appear as awarded by staff (instructor). This function will
     also mark all requirements of submitter to fulfilled, which means all ORA steps will be marked as completed.
```
Comment (ahmed-faraz46) [inline openedx/features/assessment/helpers.py:128]: Let's please update it according to the current implementation, as we are not marking all steps as completed.

**Review by ahmed-faraz46 — APPROVED** [review]
PR looks great to me 👍 Here is just one minor update required in a doc string, please cater before merging.

File: openedx/features/assessment/helpers.py (hunk 1 above)
Comment (zahid-arbisoft) [reply openedx/features/assessment/helpers.py]: Done

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Pull request merged**